routes/api_endpoints.js

```diff
@@ -0,0 +1,73 @@
+import express from 'express';
+
+const router = express.Router();
```

Comment:
Nice Job, Endeavour to create a route to delete an entry and also implement data validation

Reply:
Okay!